liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/tag.test.groovy

```diff
@@ -9,8 +9,6 @@ Short Description: Mark the current database state with the specified tag
 Long Description: NOT SET
 Required Args:
   tag (String) Tag to add to the database changelog table
-  url (String) The JDBC database connection URL
-    OBFUSCATED
 Optional Args:
```

Comment:
As this one is very simple I could not think of anything else to test. I think the only way to improve it would be writing a case that goes to the database and confirms that the tag is here, but not sure if it's worth it as other tag tests are already covering it.